Sources/LSPLogging/Logging.swift

```diff
@@ -240,3 +244,10 @@ public class AnyLogHandler: LogHandler {
     handler(message, level)
   }
 }
+
+#if canImport(os)
+@available(macOS 11.0, *)
+extension os.Logger {
+  fileprivate static let shared = os.Logger()
+}
+#endif
```

Comment:
Sorry for not mentioning this earlier, but is there a reason why we have a shared `os.Logger` and not one for each of our own `Logger` instances? If not, I think it would be cleaner if each `Logger` had its own `os.Logger`.

Reply:
I think the same at first, but the compiler won't let us do it.
It seems we can't have @available check with stored property.
https://stackoverflow.com/questions/41904724/using-available-with-stored-properties
```
public final class Logger {
#if canImport(os)
  @available(macOS 11.0, *)
  var oslogger = os.Logger() // Error
#endif
}
```